components/gitpod-protocol/src/util/tracing.ts

```diff
@@ -32,28 +32,22 @@ export namespace TraceContext {
         return opentracing.globalTracer().startSpan(operation, options);
     }
 
-    export function childContextWithSpan(operation: string, parentCtx: TraceContext): TraceContextWithSpan {
+    export function childContext(operation: string, parentCtx: TraceContext): TraceContextWithSpan {
```

Comment:
This method seems superfluous to me. In the two paces we use it I think it would be nicer just to 

```ts
const span = TraceContext.startSpan(...)
// and then use { span } when it needs to be passed to something that expects a ctx instead of a span
```

Reply:
IMO this is out-of scope here but:
 - either we keep the current style (`TraceContext`), then it's trade-off between `{ span }` on every call vs. `ctx`
 - or we switch to `span` directly
I see no pressure atm to switch styles.